src/components/Block/index.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from 'clsx'
+import { BLOCK_COLOR_CONFIG, BlockColorType } from '@/constants/block'
+
+const MAX_COLORS = 5 // 보여질 최대 색상 개수
+const HEIGHT = ['', 'h-full', 'h-1/2', 'h-1/3', 'h-1/4', 'h-1/5']
+
+interface Props {
+  colors: BlockColorType[]
+}
+
+const Block = ({ colors }: Props) => {
+  const totalBlocks = colors.length
+
+  const showColors = () => {
+    const coloredBlocks = []
+    for (let i = 0; i <= totalBlocks; i++) {
+      coloredBlocks.push(
+        <div
+          key={i}
+          className={clsx(
+            BLOCK_COLOR_CONFIG[colors[i]],
+            totalBlocks <= MAX_COLORS
+              ? HEIGHT[totalBlocks]
+              : HEIGHT[MAX_COLORS],
```

Comment:
```suggestion
            HEIGHT[Math.min(totalBlocks, MAX_COLORS)],
```

더 간단하게 수정할 수 있을 것 같아요!

Reply:
[cea3842](https://github.com/dnd-side-project/dnd-8th-7-frontend/pull/17/commits/cea3842d825309e4e15f75968a84dbdda63239d3)